documents/policies/external/Travel Reimbursement.md

```diff
@@ -1,9 +1,9 @@
 # WCA Travel Reimbursement Policy
 
-### Version 2.0 {.version}
+### Version 2.1 {.version}
 
 ## Purpose
-The purpose of this policy is to define the conditions under which the travel expenses of World Cube Association (WCA) Staff can be paid by the organization in order to further the Mission and the Spirit of the WCA.
+The purpose of this policy is to define the conditions under which the travel expenses of World Cube Association (WCA) Volunteers can be paid by the organization in order to further the Mission and the Spirit of the WCA.
```

Comment:
```suggestion
The purpose of this policy is to define the conditions under which the travel expenses of WCA Volunteers can be paid by the organization in order to further the Mission and the Spirit of the WCA.
```
Suggest avoiding definition as commonly understood and covered by Definitions Motion.